cogs/learninggroups.py

```diff
@@ -16,14 +16,14 @@
 
 """
   Environment Variablen:
-  DISCORD_LEARNINGGROUPS_OPEN - ID der Kategorie für offene Lerngruppen
-  DISCORD_LEARNINGGROUPS_CLOSE - ID der Kategorie für private Lerngruppen
-  DISCORD_LEARNINGGROUPS_ARCHIVE - ID der Kategorie für archivierte Lerngruppen
-  DISCORD_LEARNINGGROUPS_REQUEST - ID des Channels in welchem Requests vom Bot eingestellt werden
-  DISCORD_LEARNINGGROUPS_INFO - ID des Channels in welchem die Lerngruppen-Informationen gepostet/aktualisert werden
+  DISCORD_LEARNINGGROUPS_OPEN - Kategorie-ID der offenen Lerngruppen
+  DISCORD_LEARNINGGROUPS_CLOSE - Kategorie-ID der privaten Lerngruppen
+  DISCORD_LEARNINGGROUPS_ARCHIVE - Kategorie-ID der archivierten Lerngruppen
+  DISCORD_LEARNINGGROUPS_REQUEST - ID des Kanals, in dem Anfragen, die über den Bot gestellt wurden, eingetragen werden
+  DISCORD_LEARNINGGROUPS_INFO - ID des Kanals, in dem die Lerngruppen-Informationen gepostet/aktualisert werden
   DISCORD_LEARNINGGROUPS_FILE - Name der Datei mit Verwaltungsdaten der Lerngruppen (minimaler Inhalt: {"requested": {},"groups": {}})
-  DISCORD_LEARNINGGROUPS_COURSE_FILE - Name der Datei welche die Kursnamen für die Lerngruppen-Informationen enthält (minimalter Inhalt: {})
-  DISCORD_MOD_ROLE - ID der Moderator Rolle von der erweiterte Lerngruppen-Actionen ausgeführt werden dürfen
+  DISCORD_LEARNINGGROUPS_COURSE_FILE - Name der Datei welche die Kursnamen für die Lerngruppen-Informationen enthält (minimaler Inhalt: {})
+  DISCORD_MOD_ROLE - ID der Moderatorin, die erweiterte Lerngruppen-Aktionen ausführen darf

```

Comment:
Die Änderungen von 19-25 finde ich gut, müsste der Konsistenz halber auch im [.env.template](https://github.com/FU-Hagen-Discord/fernuni-bot/blob/develop/.env.template) übernommen werden.

Zeile 26: hatte ich beim letzten Review schon kommentiert (möglicherweise aber auch nur laut in meinem Kopf gesagt): die Umformulierung ändert den Sinn fundamental auf einer Art und Weise, die inhaltlich nicht stimmt. 
Es handelt sich nicht um die ID eines Mods, sondern um die ID der Mod-Rolle. Müsste also so ähnlich heißen "ID der Moderation-Rolle, die erweiterte Lerngruppen-Aktionen ausführen darf"

Reply:
Wenn du magst könnte ich mir das annehmen, allerdings bedeutet es im Rahmen dieses PR - wenn ich nichts übersehe - dass ich einen PR zu deinem Fork machen würde 😄 
Musst du sagen, was du davon hälst.

Reply:
Ahhh, Zeile 26 hatte ich so nicht verstanden. Danke. Ich habe es in "ID der Moderations-Rolle" geändert.